src/EventListener/PostQueryBuilderSubscriber.php

```diff
@@ -156,7 +167,7 @@ protected function filterQueryBuilder(QueryBuilder $queryBuilder, string $field,
             // Multiple values leads to IN statement
             case $value instanceof Collection:
             case \is_array($value):
-                if (0 < count($value)) {
+                if (0 < \count($value)) {
```

Comment:
I'm gonna push that straight to master. Thanks

Reply:
I'm pretty sure PHPCSFixer did it on its own, but you're welcome

Reply:
I'm interested to know the PHP-CS parameter that handle that !

Reply:
do you talk about adding "\"? its "native_function_invocation"

Reply:
ok, it's already in the bundle PHP-CS config. It's been added by a contributor.